src/app/communities/communities.component.ts

```diff
@@ -135,4 +140,19 @@ export class CommunitiesComponent implements OnInit {
     this.changeDetector.markForCheck();
   }
 
+  private showInfoDialog(community: CommunityResponse) {
+    this.dialog.open(ClosedCommunityInfoDialogComponent, {
+      width: '350px',
+      data: <CommunityResponse>{
```

Comment:
Can we just pass the community object into ClosedCommunityInfoDialogComponent without copying it? Or at least use `Object.assign(target, source)` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) or [the spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals)?

Reply:
done